### ISSUE TITLE
License field no longer contains "title"

### ISSUE THREAD

**Issue by AlmightyYakob** [dandi/dandi-cli#298]
The relevent [section](https://github.com/dandi/schema/blob/26bb8e1f72927d681f5ffa293f4c44c34083ab36/releases/1.0.0-rc1/dandiset.json#L125-L130) from the generated dandiset.json:

```
    "license": {
      "nskey": "schema",
      "type": "array",
      "items": {
        "$ref": "#/definitions/LicenseType"
      }
```

Is there a reason this was removed? All top level fields should have a `title` field.

**Comment by satra** [comment]
@AlmightyYakob - i see it in my generated version. let me check the repo and the code that's generating it.

**Comment by satra** [comment]
this is due to changes in pydantic. will update the models.

**Comment by yarikoptic** [comment]
We should ideally add the check for it in the tests I guess?

**Comment by satra** [comment]
i don't know what is happening yet. `license` seems to be very special :) still debugging.

**Comment by satra** [comment]
should be addressed in #299 after my PR to jake's PR is merged.

**Comment by AlmightyYakob** [comment]
Fixed via #299
